### DISCUSSION TITLE
Add Claude 3.5 Sonnet New or at east model name mapping.

### DISCUSSION THREAD

**Discussion by adamholter** [cline/cline#594] [Feature Requests]
Add Claude 3.5 Sonnet New or at east model name mapping.
![image](https://github.com/user-attachments/assets/ffd592b4-f1c1-4ea9-9813-d1eb7e269c99)
Would be cool if I could put the model name in manually.

**Comment by kcui23** [comment]
Yes, we definitely need it! By the way, OpenRouter has already updated to the 1022 API, so that’s available now. 😊

**Comment by PhunkyBob** [comment]
Solved with 165272a.
